daffodil-lib/src/main/resources/org/apache/daffodil/xsd/tdml.xsd

```diff
@@ -349,6 +349,7 @@
         <xs:restriction base="xs:string">
           <xs:enumeration value="all"/>
           <xs:enumeration value="any"/>
+          <xs:enumeration value="none"/>
```

Comment:
I didn't even know this "match" attribute existed. I guess we never implemented it?

I'm wonder if it would be useful to have something a big more complex? For example, I could imagine a test might want to do something like diags "a", "b", and "c", must all appear, one of "m" or "n" must apear, and none of "x", "y", and "z" must appear. Maybe we need to allow muptilpe <tdml:error> tags? Then this would be described like

```xml
<parserTestCase ...>
  ... 
  <errors match="all">
    <error>a</error>
    <error>b</error>
    <error>c</error>
  </errors>
  <errors match="any">
    <error>m</error>
    <error>n</error>
  </errors>
  <errors match="none">
    <error>x</error>
    <error>y</error>
    <error>z</error>
  </errors>
</parserTestCase>
```
Maybe that's unneeded complexity though?